daemon/daemon.go

```diff
@@ -870,7 +876,10 @@ func NewDaemonFromDirectory(config *Config, eng *engine.Engine) (*Daemon, error)
 
 	daemonRepo := path.Join(config.Root, "containers")
 
-	if err := os.MkdirAll(daemonRepo, 0700); err != nil && !os.IsExist(err) {
+	if err := os.MkdirAll(daemonRepo, 0711); err != nil && !os.IsExist(err) {
```

Comment:
Can you explain the change of permissions? Is this to allow the non-root docker user to cd into this dir?